app/src/main/java/net/squanchy/onboarding/account/AccountOnboardingActivity.kt

```diff
@@ -0,0 +1,101 @@
+package net.squanchy.onboarding.account
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import io.reactivex.android.schedulers.AndroidSchedulers
+import kotlinx.android.synthetic.main.activity_onboarding_account.onboardingContentRoot
+import kotlinx.android.synthetic.main.activity_onboarding_account.onboardingSignInButton
+import kotlinx.android.synthetic.main.activity_onboarding_account.onboardingSkip
+import net.squanchy.R
+import net.squanchy.fonts.TypefaceStyleableActivity
+import net.squanchy.navigation.Navigator
+import net.squanchy.onboarding.Onboarding
+import net.squanchy.onboarding.OnboardingPage
+import net.squanchy.signin.SignInService
+import java.util.concurrent.TimeUnit
+
+class AccountOnboardingActivity : TypefaceStyleableActivity() {
+
+    private lateinit var onboarding: Onboarding
```

Comment:
I'm not really a fun of `lateinit var`, we could make this `val` easily by creating an extension function like:

```
fun <T> unsafeLazy(initialiser: () -> T) = lazy(LazyThreadSafetyMode.NONE, initialiser)
```

and then it would be:
```
private val component by unsafeLazy { AccountOnboardingInjector.obtain(this) }
private val onboarding by unsafeLazy { component.onboarding() }
private val navigator by unsafeLazy { component.navigator() }
private val signInService by unsafeLazy { component. signInService() }
```

Reply:
I'm not really sure I like this more to be honest, it seems just a way to mask a non-problem with something that we don't really need.

Reply:
Yeah that's true, to me it's just a way to define something that should be `final` but is not because of the Android lifecycle. But it's not really a problem